.github/workflows/velox_backend.yml

```diff
@@ -95,7 +95,15 @@ jobs:
             mkdir -p /work/.m2/repository/org/apache/arrow/
             cp -r /root/.m2/repository/org/apache/arrow/* /work/.m2/repository/org/apache/arrow/
           "
-
+          # check release files
```

Comment:
This job will fail when velox or gluten build fails, see: https://github.com/apache/incubator-gluten/actions/runs/13914275514/job/38934362713?pr=9028